v3/examples/theme/assets/style.css

```
@@ -0,0 +1,58 @@
+.btn {
+    padding: 5px 10px;
+}
+
+.wrapper {
+    display: flex;
+    flex-direction: row;
+    width: 100vw;
+    height: 100vh;
+    gap: 1px;
+
+    background-color: var(--secondary-color);
```

Stylelint (17.11.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)